### PR TITLE
Bump version to 0.6.0.0

### PR DESCRIPTION
**Pull request by jhrcek**
@wz1000 I'm not really sure if we want to bump to 0.6.0.0 or just 0.5.2.0.

### CODE REVIEW

**Comment by wz1000** [comment]
0.6.0.0 should be fine as there were breaking changes.

**Review by wz1000 — APPROVED** [review]

**Comment by jhrcek** [comment]
Ok, pushed a tag 0.6.0.0. I'll upload it to hackage once you merge (I don't have permissions to do that for this repo).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
 # Revision history for hiedb
 
+## 0.6.0.0 -- 2024-12-11
```
Comment (josephsumabat) [inline CHANGELOG.md:3]: I think this was supposed to be 2024-02-11?

Comment (jhrcek) [reply CHANGELOG.md:3]: Oops, you're right. I'll push a fix

**Pull request merged**